info.femm.unofficial-manifest.yml

```diff
@@ -0,0 +1,235 @@
+app-id: info.femm.unofficial
+base-version: '21.08'
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: femm.sh
+build-options:
+  env:
+    - "WINEPREFIX": "/var/data/.local/share/femm42-flatpak"
```

Comment:
I don't think this is right in any way, shape, or form.

Reply:
Me neither. I need suggestion and help.